pkgs/development/tools/yo/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "yo";
+  version = "4.3.1";
+
+  src = fetchFromGitHub {
+    owner = "yeoman";
+    repo = "yo";
+    rev = "v${version}";
+    hash = "sha256-vnvcg3hvAYcqS11enBEHtpTwTOy4puY5i/6zPOHCywo=";
+  };
+
+  npmDepsHash = "sha256-QkEPaepvI6NfEEmqnVA4Xx/tByn6goyGWVpoJNMigd8=";
+
+  dontNpmBuild = true;
+
+  meta = {
+    description = "CLI tool for running Yeoman generators";
+    homepage = "https://github.com/yeoman/yo";
+    license = lib.licenses.bsd2;
+    mainProgram = "yo";
+    maintainers = with lib.maintainers; [ ma27 ];
```

Comment:
fwiw I haven't used this in a long time and if the old subsystem had maintainer fields, I would've removed myself from there a while ago already. I'm not sure if that tool is still relevant (i.e. whether or not we want to keep this), but please don't list me as maintainer here.